[2025-06-01T11:40:29.035+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: movie_recommendation_pipeline.predict_best_model manual__2025-06-01T11:34:46.516239+00:00 [queued]>
[2025-06-01T11:40:29.042+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: movie_recommendation_pipeline.predict_best_model manual__2025-06-01T11:34:46.516239+00:00 [queued]>
[2025-06-01T11:40:29.043+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-06-01T11:40:29.053+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): predict_best_model> on 2025-06-01 11:34:46.516239+00:00
[2025-06-01T11:40:29.061+0000] {standard_task_runner.py:60} INFO - Started process 871 to run task
[2025-06-01T11:40:29.062+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'movie_recommendation_pipeline', 'predict_best_model', 'manual__2025-06-01T11:34:46.516239+00:00', '--job-id', '993', '--raw', '--subdir', 'DAGS_FOLDER/mlops_movie_dag.py', '--cfg-path', '/tmp/tmp4jo6y8_7']
[2025-06-01T11:40:29.064+0000] {standard_task_runner.py:88} INFO - Job 993: Subtask predict_best_model
[2025-06-01T11:40:29.100+0000] {task_command.py:423} INFO - Running <TaskInstance: movie_recommendation_pipeline.predict_best_model manual__2025-06-01T11:34:46.516239+00:00 [running]> on host 754694dd6447
[2025-06-01T11:40:29.164+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='movie_recommendation_pipeline' AIRFLOW_CTX_TASK_ID='predict_best_model' AIRFLOW_CTX_EXECUTION_DATE='2025-06-01T11:34:46.516239+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-01T11:34:46.516239+00:00'
[2025-06-01T11:41:35.954+0000] {mlops_movie_dag.py:13} INFO - 2025-06-01 11:40:30,731 - INFO - 🚀 Starte Vorhersage mit MLflow Registry-Modell
2025-06-01 11:40:31,163 - INFO - 📥 User-Matrix geladen: /opt/***/data/processed/user_matrix.csv – Shape: (138493, 20)

Downloading artifacts:   0%|          | 0/1 [00:00<?, ?it/s]
Downloading artifacts:   0%|          | 0/1 [01:03<?, ?it/s]
2025-06-01 11:41:35,759 - ERROR - ❌ Kein Modell mit Alias @best_model gefunden! Prüfe, ob das Modell korrekt registriert und der Alias gesetzt ist. Error: The following failures occurred while downloading one or more artifacts from https://dagshub.com/sacer11/MLOps_movie_rec_apr25.mlflow/api/2.0/mlflow-artifacts/artifacts/b2cca28f98b24ce191a79aad3f1695f1/08995a6854df4d62ad87a673bafbc0fd/artifacts/hybrid_knn_pyfunc:
##### File  #####
API request to https://dagshub.com/sacer11/MLOps_movie_rec_apr25.mlflow/api/2.0/mlflow-artifacts/artifacts/b2cca28f98b24ce191a79aad3f1695f1/08995a6854df4d62ad87a673bafbc0fd/artifacts/hybrid_knn_pyfunc/ failed with exception HTTPSConnectionPool(host='dagshub.com', port=443): Max retries exceeded with url: /sacer11/MLOps_movie_rec_apr25.mlflow/api/2.0/mlflow-artifacts/artifacts/b2cca28f98b24ce191a79aad3f1695f1/08995a6854df4d62ad87a673bafbc0fd/artifacts/hybrid_knn_pyfunc/ (Caused by ResponseError('too many 500 error responses'))

[2025-06-01T11:41:35.956+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-01T11:41:35.967+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=movie_recommendation_pipeline, task_id=predict_best_model, execution_date=20250601T113446, start_date=20250601T114029, end_date=20250601T114135
[2025-06-01T11:41:36.007+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-01T11:41:36.024+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
